stdlib/Dates/docs/src/index.md

```diff
@@ -579,6 +579,26 @@ julia> Dates.value(Dates.Millisecond(10))
 10
 ```
 
+Representing periods or durations that are not integer multiples of the basic types can be
+achieved with the [`CompoundPeriod`](@ref) type. Compound periods may be constructed manually from simple
```

Comment:
The tests are throwing an error saying there are no docs to cross-reference for `CompoundPeriod`; would you mind adding some or at least taking a look at that? Otherwise, I think this PR looks great!

Reply:
`CompoundPeriod` isn't exported, it's fixed now.